cli/cli_state.ml

```diff
@@ -495,6 +497,7 @@ let activate_contact state active =
         active_contact      = njid ;
         scrollback          = 0 ;
         filter              = None ;
+        hide_presence       = false ;
```

Comment:
well, this line makes it reset when you go to another contact... it's not entirely clear to me what the desired behaviour is... i guess it could even be worth to persist this on disk for each muc (it's really only relevant in muc, no?)

Reply:
Yes, I was trying to have it behave like `/filter` without realizing how `/filter` actually behaves. I think it's only MUCs where this is relevant / useful. IIRC presence updates from users go into your own contact "chat".

I don't have strong feelings on how it should behave regarding persistence, but I'm open to implement whatever you think is the best.

Reply:
well, you're the one who wants that feature ;) so you should propose what is most convenient for you! :)

Reply:
I've heard people complain about "people coming and going" spam annoying users, so I think a general one (for your own contact tab) would make sense too.

Reply:
This covers also your own contact tab btw.